src/lib/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js

```diff
@@ -269,6 +288,13 @@ class PatronCurrentDocumentRequests extends Component {
           )}
           <ILSItemPlaceholder fluid isLoading={isLoading}>
             <Error error={error}>
+              {documentRequests.hits.length > 2 && this.renderPagination()}
```

Comment:
not sure why if there is more than 2 hits then the pagination is showed ? why specifically 2 ?

Reply:
Having both pagination at the top and at the bottom makes it look ugly when there are less than three items in the list (both pagination elements are far to close to each other). I decided to not show the top one with two items or less.

Reply:
See my comment to avoid to use `.length`.
I think there is an issue with the logic of your pagination cmp then: it should not render if only one page.

Reply:
It doesn't.
The problem this condition is solving is not if there is only page but when there are multiple pages and one of them (the last one) as to few rows.

Reply:
so if you are on the last page, and the last page has only one result, then the pagination disappears ? do I understand correctly? If yes, then I would prefer to leave both paginations, since it will prevent the UI from jumping and suddenly changing many things. I think the actionable buttons, as pagination should not change so suddenly, since it can be confusing for the user, who might not understand why suddenly one of the paginations disappeared

Reply:
It looks a bit weird I have to say. I will share a screenshot after pushing again.